### PR TITLE
Changes for switching from dbnode4 to new IAM data

### PR DESCRIPTION
**Pull request by sapnamysore**
This branch is deployed on dev

### CODE REVIEW

File: qualtrics_link/util.py [hunk 1]
```diff
@@ -440,9 +443,8 @@ def get_person_details(huid, person_list=None):
         valid_school = True
         division = valid_school_code
 
-    # Department Affiliations check
-    if person.department != '':
-        valid_dept_name = get_valid_dept(person.department)
+    if person.role_type_cd.lower() == 'employee':
+        valid_dept_name = get_valid_dept(person.faculty_cd)  # Change this to department when looking at the OldPeople model
```
Comment (cmsnowden) [inline qualtrics_link/util.py]: this comment can go

Comment (sapnamysore) [reply qualtrics_link/util.py]: Removed

**Comment by cmurtaugh** [comment]
Looks good to me -- I'd consider upgrading Django to the latest 1.11 version as well

**Comment by sapnamysore** [comment]
Upped Django version and merging

**Pull request merged**